pmda/util.py

```diff
@@ -223,6 +256,19 @@ def second_order_moments(S1, S2):
     return S
 
 
-def sumofsquares(array):
-    """Calculates the sum of squares"""
-    return np.sum((array - array.mean(axis=0))**2, axis=0)
+def fold_second_order_moments(*args):
+    """
+    'Fold' action for second_order_moments calculation.
```

Comment:
What does "fold" mean? Explain what it does, i.e., answer the question "what do I need this function for?"

Reply:
If this is a "reduce" function then you can say this. However, remember that a user does not care too much about the implementation (application of reduce) but what it does (effectively summing 1st and 2nd moments from blocks).